go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/moby/buildkit v0.6.2 // indirect
 	github.com/opencontainers/runc v1.0.0-rc9
 	github.com/opencontainers/runtime-spec v1.0.0
-	github.com/openshift/api v3.9.1-0.20191024142031-e89ff1fddcbe+incompatible
+	github.com/openshift/api v0.0.0-20200618202633-7192180f496a
```

Comment:
Where did this api bump come from? This appears to be from a later OpenShift release.

Reply:
That is the latest commit in the release-4.4 branch of the openshift/api repository: https://github.com/openshift/api/commit/7192180f496aab1f7659d8660fc360498bab498b

Reply:
Please indicate in the commit message that you are bumping openshift/api, and the reason for doing so. I assume you did this to get around an "invalid pseudo-version" error? I found something similar in a separate item I was iterating on.